src/command_modules/azure-cli-resource/tests/test_custom.py

```diff
@@ -0,0 +1,49 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import unittest
+
+from azure.cli.command_modules.resource.custom  import _merge_parameters
+
+class TestCustom(unittest.TestCase):
+    
+    @classmethod
+    def setUpClass(cls):
+        pass
+
+    @classmethod
+    def tearDownClass(cls):
+        pass
+
+    def setUp(self):
+        pass
+
+    def tearDown(self):
+        pass
+
+
+    def test_merge_parameters(self):
```

Comment:
Please add a more accurate name. Due to the current test layout, you'd better have the test method name to be unique so as to facilitate the parallel test run. For example, this method can be named to `test_resource_merge_parameters`.